generated/nidcpower/session.py

```diff
@@ -1625,13 +1627,17 @@ def fetch_multiple(self, count, timeout=1.0):
         channel_name_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case C010
         timeout_ctype = visatype.ViReal64(timeout)  # case S150
         count_ctype = visatype.ViInt32(count)  # case S190
-        voltage_measurements_ctype = (visatype.ViReal64 * count)()  # case B600
-        current_measurements_ctype = (visatype.ViReal64 * count)()  # case B600
+        voltage_measurements_size = count  # case B600
```

Comment:
I did a little bit of experimentation that made me question our use of array.array.

Specifically: The main thing we get by using array.array is that we no longer copy elements into a list. How would that be different than returning the object we get from `(visatype.ViReal64 * count)()`, which would be of type `<class '__main__.c_double_Array_10'>` rather than `<type 'array.array'>`?

Very little.

* In both cases the data is zeroed out so that's a shame and the advantage of numpy.
* In both cases the object the user gets back can be indexed.
* In both cases the object's size cannot be changed.
* array.array.__str__() is more useful.

Is that it?
Does it warrant the increased complexity in the generator?

Reply:
ctype arrays don't support memoryview like an `array.array` does. You can create the memoryview but then you can't slice it.